source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -1,101 +1,116 @@
 .. _mixin:
 
-Create and add custom bundles to your upstream Clear Linux system
-#################################################################
+Mixin tool
```

Comment:
Wonderful revision to the title. Finally, some consistency and brevity!

Reply:
😃